src/modules/decklink/producer_decklink.yml

```diff
@@ -113,3 +113,12 @@ parameters:
       It skips frames that has VITC timecode less then specified.
       After reaching first frame with timecode greater or equal then specified this
       property is reset to zero.
+
+  - identifier: 10
```

Comment:
yeah, I did not like "10" when I saw it. I was going to suggest "10bit". It is not good behavior to add new image format conversion code into a producer in order to comply with a requested image format unless it already has something at its disposal (for example, avformat producer can use libswscale to comply). So, mlt_image_format does not seem correct to me.